### 1. Из колоды в 52 карты извлекаются случайным образом 4 карты. 

#### a) Найти вероятность того, что все карты – крести.

In [31]:
print(f'Вероятность достать первую крестовую карту 13/52, вторую 12/51, третью 11/50, четвертую 10/49.\n'
      f'Искомая вероятность: {13/52*12/51*11/50*10/49}')

Вероятность достать первую крестовую карту 13/52, вторую 12/51, третью 11/50, четвертую 10/49.
Искомая вероятность: 0.0026410564225690276


Ответ: 0.0026410564225690276

_______________________________________________________________________________________________________________________________

####  б) Найти вероятность, что среди 4-х карт окажется хотя бы один туз.

In [34]:
import math  
n=math.factorial(52)/(math.factorial(4)*(math.factorial(52 - 4)))
m=math.factorial(48)/(math.factorial(4)*(math.factorial(48 - 4)))
print(f'Общее число доступных комбинаций: {n}')
print(f'Общее число комбинаций в которых не будет туза: {m}')
print(f'Вероятность, что в выбранных картах не будет туза: {m/n}')
print(f'Вероятность, что в выбранных картах будет хотя бы один туз: {1-m/n}')

Общее число доступных комбинаций: 270725.0
Общее число комбинаций в которых не будет туза: 194580.0
Вероятность, что в выбранных картах не будет туза: 0.7187367254594146
Вероятность, что в выбранных картах будет хотя бы один туз: 0.2812632745405854


Ответ: 0.2812632745405854

_______________________________________________________________________________________________________________________________

### 2. На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. Код содержит три цифры, которые нужно нажать одновременно. Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?

In [36]:
n=math.factorial(10)/(math.factorial(3)*(math.factorial(10 - 3)))
print(f'Общее число доступных комбинаций: {n}')
print(f'Искомая вероятность: {1/n}')

Общее число доступных комбинаций: 120.0
Искомая вероятность: 0.008333333333333333


Ответ: 0.008333333333333333

_______________________________________________________________________________________________________________________________

### 3. В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали. Какова вероятность того, что все извлеченные детали окрашены?

In [37]:
print(f'Вероятность достать первую окрашенную деталь 9/15, вторую 8/14, третью 7/13.\n'
      f'Искомая вероятность: {9/15*8/14*7/13}')

Вероятность достать первую окрашенную деталь 9/15, вторую 8/14, третью 7/13.
Искомая вероятность: 0.1846153846153846


Ответ: 0.1846153846153846

______________________________________________________________________________________________________________________________

### 4. В лотерее 100 билетов. Из них 2 выигрышных. Какова вероятность того, что 2 приобретенных билета окажутся выигрышными?

In [39]:
print(f'Вероятность что первый билет будет выйгрышным 2/100, а второй 1/99.\n'
      f'Искомая вероятность: {2/100*1/99}')

Вероятность что первый билет будет выйгрышным 2/100, а второй 1/99.
Искомая вероятность: 0.00020202020202020202


Ответ: 0.00020202020202020202